dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -201,6 +201,10 @@
     @CollectionTable(uniqueConstraints = @UniqueConstraint(name = "unique_paths", columnNames = { "entry_id", "filetype", "path" }))
     private Map<DescriptorLanguage.FileType, String> defaultPaths = new HashMap<>();
 
+    @Column
+    @ApiModelProperty(value = "The Digital Object Identifier (DOI) representing all of the versions of your workflow", position = 13)
```

Comment:
I wonder how positions work in a base class. 
Does 13 overlap with any of the positions in the children?

Reply:
Not sure if anyone's noticed, but for Tag/WorkflowVersion, and Version, I made the Version start numbering at 1 and made Tag/WorkflowVersion start numbering at 101

Reply:
(didn't notice)